test/views/sections/items/ItemsAddToListViewSpec.scala

```diff
@@ -0,0 +1,186 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views.sections.items
+
+import base.SpecBase
+import fixtures.messages.sections.items.ItemsAddToListMessages
+import fixtures.{DocumentTypeFixtures, ItemFixtures}
+import forms.sections.items.ItemsAddToListFormProvider
+import mocks.services.MockGetCnCodeInformationService
+import models.UnitOfMeasure.{Litres20, Thousands}
+import models.requests.{CnCodeInformationItem, DataRequest}
+import models.response.referenceData.CnCodeInformation
+import org.jsoup.Jsoup
+import org.jsoup.nodes.Document
+import org.scalatest.concurrent.IntegrationPatience
+import pages.sections.items._
+import play.api.i18n.Messages
+import play.api.mvc.AnyContentAsEmpty
+import play.api.test.FakeRequest
+import uk.gov.hmrc.http.HeaderCarrier
+import viewmodels.checkAnswers.sections.items.ItemPackagingSummary
+import viewmodels.helpers.ItemsAddToListHelper
+import views.html.components.{span, tag}
+import views.html.sections.items.ItemsAddToListView
+import views.{BaseSelectors, ViewBehaviours}
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class ItemsAddToListViewSpec extends SpecBase with ViewBehaviours with DocumentTypeFixtures
+  with IntegrationPatience with ItemFixtures with MockGetCnCodeInformationService {
```

Comment:
Not sure if IntegrationPatience is needed?

```suggestion
   with ItemFixtures with MockGetCnCodeInformationService {
```